cmd/access/node_builder/access_node_builder.go

```diff
@@ -746,7 +746,7 @@ func (builder *FlowAccessNodeBuilder) BuildExecutionSyncComponents() *FlowAccess
 			)
 
 			r, err := edrequester.New(
-				builder.Logger,
+				builder.Logger.With().Str("entity", "execution_data").Logger(),
```

Comment:
We don't need the change here because this is already a type-specific requester engine type.